avocado/utils/cpu.py

```diff
@@ -550,6 +550,23 @@ def wrap(*args, **kwargs):
     return wrap
 
 
+def lscpu():
+    # Getting the values from lscpu
+    output = process.run("lscpu")
+    # res list contains values as : Cores , Physical Sockets , Physical Chips , Chips
+    res = []
+    for line in output.stdout.decode("utf-8").split('\n'):
+        if 'Core(s) per socket:' in line:
+            res.append(int(line.split(':')[1].strip()))
+        if 'Physical sockets:' in line:
+            res.append(int(line.split(':')[1].strip()))
+        if 'Physical chips:' in line:
+            res.append(int(line.split(':')[1].strip()))
+    # chip = physical socket x physical chip
+    res.append(int(res[1]*res[2]))
```

Comment:
When the lscpu doesn't return 'Physical sockets:' or 'Physical chips:' this will raise an Index Error.